### PR TITLE
Upgraded openapi-reader to support YAML

### PR DESCRIPTION
**Pull request by francescozanoni**
Hi, man.
I've upgraded openapi-reader constraint, in order to add support for OpenAPI schema in YAML format.
I haven't added any further tests, since I think they would only test the project dependency: am I wrong?
Thanks in advance.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,8 +12,9 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-json": "*",
         "ckr/arraymerger": "^2.0",
-        "hkarlstrom/openapi-reader": "^0.2.1",
+        "hkarlstrom/openapi-reader": "~0.2",
```
Comment (hkarlstrom) [inline composer.json]: To my understanding `~0.2` means>=0.2 and < 1.0.0. As the YAML support was added to the reader in 0.3, this should be `^0.3`?

Comment (francescozanoni) [reply composer.json]: Yes, you're right: I wasn't considering the whole situation.

Comment (hkarlstrom) [reply composer.json]: Im not sure either, but I think you can just push to your branch and the pull request will be automatically updated. If it's not, just create a new one.

Comment (francescozanoni) [reply composer.json]: Ummm, yep, I think it's automatic: I can see my commit already in the list of this pull request.

**Pull request merged**